src/StructureFactors.jl

```diff
@@ -427,8 +443,9 @@ function fft_spin_traj!(spin_traj::Array{ComplexF64};
                         plan::Union{Nothing, FFTW.cFFTWPlan}=nothing)
     if isnothing(plan)
         FFTW.fft!(spin_traj, (2,3,4,6))
+        spin_traj /= N
     else
-        spin_traj = plan * spin_traj
+        spin_traj = (plan * spin_traj) ./ 1e11
```

Comment:
1e11 = inf?

Reply:
This is strange. I don't think this function is ever called. Only the one above it. I'll remove it.

Reply:
I guess it is called... I'll figure out what's going on.